### DISCUSSION TITLE
Working Docker Compose?

### DISCUSSION THREAD

**Discussion by gerroon** [kiwix/kiwix-tools#656] [Q&A] [answered]
Hi

Does anyone have a properly setup DC setup for one click install and serve over the web?

thanks

**Comment by kelson42** [comment, marked as answer]
What is wrong with https://github.com/kiwix/kiwix-tools/tree/main/docker/server ?

**Reply by gerroon** [reply]
I did not see that one, the one I saw on the Docker hub just a single docker run ghcr.io/kiwix/kiwix-serve  type thing where it was not clear about how to download or serve, where to save the data etc.
